test/RoadRegistry.Tests/Framework/Projections/WmsContextScenarioExtensions.cs

```diff
@@ -0,0 +1,137 @@
+namespace RoadRegistry.Framework.Projections
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.Connector;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.Connector.Testing;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.SqlStreamStore;
+    using KellermanSoftware.CompareNetObjects;
+    using Microsoft.EntityFrameworkCore;
+    using Wms.Schema;
+    using Xunit.Sdk;
+
+    internal class MemoryWmsContext : WmsContext {
```

Comment:
Not sure this is needed (was needed for Product / Editor to override Query Types).

Reply:
Not sure either, left it there to possibly remove later.

Reply:
Resolved.